### ISSUE TITLE
Sound Classifier: how to load audio file to test model 

### ISSUE THREAD

**Issue by jamois** [apple/turicreate#1696]
Trying out the new sound classifier but can't seem to find docs or examples on how to test a sound model with, say, a single .wav file.  I created the sound model by following the example (👍🏼) but what I really need is something similar to what I use for testing my image classification models in python, e.g.

```
model = tc.load_model(modelFolder)
test = tc.SFrame({'image': [tc.Image(imagefile)]})
test['predictions'] = model.predict(test)
```

I also found the turicreate.Image class and expected a turicreate.Sound class but did not find one so I'm obviously not on the right path.  Hoping for some help.  Thx!

**Comment by TobyRoseman** [comment]
@jamois - Thanks for trying the Sound Classifier.

I agree it's confusing that we have a `tc.Image` but no `tc.Sound`. I've created #1697 to specifically track that issue. 

You can use `load_audio(...)` to load a single wav file. So you can just do this:
```python
model = tc.load_model(modelFolder)
test = tc.load_audio(imagefile)
test['predictions'] = model.predict(test)
```

Let us know if you have any other questions/feedback/comments about the sound classifier.

**Comment by jamois** [comment]
Hey @TobyRoseman Thanks for the quick response... I did just that:

```
model = tc.load_model(modelFolder)
test = tc.load_audio(audiofile)
test['predictions'] = model.predict(test)
```
and while it "worked" the predictions are pretty bad.  The model I'm using is the ESC-50 as explained here: [(https://apple.github.io/turicreate/docs/userguide/sound_classifier/)].  After many failed attempts where I submitted audio files that were not part of the training set, I decided to try submitting a file that *was* part of the set because I assumed it would produce an accurate prediction.  However, almost none of the files submitted produce an accurate prediction

```
files: 1-13572-A-46.wav & 4-125825-B-46.wav
actual: "church bells" 
prediction: "rooster"
```

Is it simply that the ESC-50 data set does not contain enough samples per class/category, e.g. there are not enough rooster sound clips to accurately recognize a rooster?  Note, the accuracy value produced at the end of training was only 0.84 (using default iterations) which helps explain some of what I am seeing, I suppose.  What about transfer learning?  I don't see a parameter to include a pretrained model like what is provided for an image classifier.

Thanks!

**Comment by TobyRoseman** [comment]
The Sound Classifier does use transfer learning. That is [2nd stage of the process](https://apple.github.io/turicreate/docs/userguide/sound_classifier/how-it-works.html#vggish-feature-extraction-stage). There is no parameter for the pretained model because we currently only support one pertained model (VGGish).

Are you getting 84% accuracy on the train set or the test set? If it's on the train set try increasing the value of the `max_iteration` parameter in the call to `tc.sound_classifier.create(...)`. 

In general, I would say sound classification is currently a harder problem than image classification. Also the ESC-50 dataset is not very large. Take a look at the [ESC-50 results](https://github.com/karoldvl/ESC-50#results). The highest reported results are only 86.5%. Human level accuracy is only 81.3%.

Also one of the two files you mention (`1-13572-A-46.wav`) is in the first fold. So using the example code that will actually be part of the test set; although that file is part of the ESC-50, it's new to your model.

I am working on features which should allow you to get better accuracy (#1477, #1571). Let me know if you have any ideas.

**Comment by jamois** [comment]
@TobyRoseman Previously I tried 5000 when I reported the 84%.  But it sounds like more data and more iterations are what I should try next.  Glad you are working on improving accuracy!

Thanks!

**Comment by TobyRoseman** [comment]
I'm going to close this issue but let us know if you have any additional questions. 

We're tracking the primary issue discussed here in #1697.
